File: src/bundesrag/locales/en.py
MESSAGES = {
    "answer_must_be_integer": "The answer must be an integer.",
    "app_help": "Download Bundestag documents and ask questions about them.",
    "ask_help": "Answers QUESTION based on the stored documents.",
    "ask_query_feedback": "What should be adjusted about the query?",
    "clear_help": "Deletes all downloaded documents and resets the vector store.",
    "ask_download_count": (
        "{num_matched} documents found, {num_existing} of them already downloaded. "
        "How many of the remaining {num_to_download} should be downloaded "
        "(most recent first, Enter for all, 0 to cancel)? "
    ),
    "clear_yes_option_help": "Delete without asking for confirmation.",
    "confirm_delete_all_yn": (
        "Really delete all downloaded documents and reset the vector store? [y/N] "
    ),
    "confirm_use_query_yn": "Use this query? [y/N] ",
    "confirmation_required": "Confirmation required.",
    "delete_done": "Done: {num_files} files deleted and vector store reset.",
    "document_reference": "Document/protocol {dokumentnummer}",
    "download_aborted": "Aborted: download of {count} documents cancelled.",
    "download_done": "Done: {num_documents} documents downloaded.",
    "download_help": (
        "Downloads documents matching PROMPT, without indexing them.\n"
        "\n"
        "PROMPT is a natural-language description of the documents to fetch. "
        "An LLM translates it into DIP API filters and asks for clarification "
        "if the prompt is too vague.\n"
        "\n"
        "Available endpoints:\n"
        "\n"
        "\b\n"
        "- drucksache: motions, bills, minor interpellations, etc.\n"
        "- plenarprotokoll: plenary session protocols.\n"
        "\n"
        "Available filters (all optional):\n"
        "\n"
        "\b\n"
        '- datum_start / datum_end: date range (e.g. "since 2026-01-01")\n'
        "- wahlperiode: electoral term number (e.g. 21)\n"
        '- dokumentnummer: exact document/protocol number (e.g. "19/1")\n'
        "- zuordnung: BT, BR, BV, or EK\n"
        '- drucksachetyp: document type, e.g. "Antrag", "Gesetzentwurf" (drucksache only)\n'
        '- urheber: originator/parliamentary group, e.g. "Fraktion der SPD" (drucksache only)\n'
        "- ressort_fdf: lead federal ministry (drucksache only)\n"
        "- titel: search terms in the title, OR-combined (drucksache only)\n"
        "\n"
        "Note: urheber and ressort_fdf use AND logic across multiple values. "
        "To find documents from either of two authors, run separate download "
        "commands."
    ),
    "download_partial_failure": (
        "Warning: {num_failed} document(s) could not be downloaded and were skipped."
    ),
    "download_skipped_existing": (
        "Note: {num_skipped} document(s) were already downloaded and were skipped."
    ),
    "filter_dokumentnummer": "  Document number: {value}",
    "filter_drucksachetyp": "  Document type: {value}",
    "filter_ressort": "  Department: {value}",
    "filter_titel": "  Title contains: {value}",
    "filter_urheber": "  Originator: {value}",
    "filter_wahlperiode": "  Electoral term: {value}",
    "filter_zeitraum": "  Date range: {start} to {end}",
    "filter_zuordnung": "  Assignment: {value}",
    "index_counts": "{num_to_index} document(s) to index, {num_indexed} already indexed.",
    "index_done": "Done: {num_documents} documents, {num_chunks} chunks stored.",
    "index_help": "Indexes previously downloaded but not yet indexed documents.",
    "job_not_cancellable": "The job is no longer running and cannot be cancelled.",
    "job_not_found": "Job not found.",
    "job_not_waiting": "The job is not waiting for input.",
    "operation_cancelled": "Operation cancelled.",
    "page_short": "p. {page}",
    "page_suffix": ", page {page}",
    "planned_query_header": "Planned DIP query (endpoint: {endpoint}):",
    "progress_step": "[Step {n}/{total}] {name}",
    "query_agent_failed": (
        "Could not build a valid DIP query from the request even after "
        "several clarification rounds."
    ),
    "serve_help": "Starts the local web interface (FastAPI + Vue SPA).",
    "serve_host_option_help": "Bind address of the web server.",
    "serve_port_option_help": "Port of the web server.",
    "serve_reload_option_help": "Auto-reload on source changes (development only).",
    "serve_started": "Web interface running at http://{host}:{port}/ (stop with Ctrl+C).",
    "similarity_suffix": " (similarity: {score})",
    "sources_header": "\nSources:",
    "status_file_indexed": "indexed",
    "status_file_not_indexed": "not indexed",
    "status_files_header": "Files:",
    "status_help": "Shows how many documents are downloaded and indexed.",
    "status_num_chunks": "Chunks in the vector store: {count}",
    "status_num_downloaded": "Downloaded: {count}",
    "status_num_indexed": "Indexed: {count}",
    "status_pdf_size": "Disk usage of PDFs: {size}",
    "status_vectorstore_size": "Disk usage of the vector store: {size}",
    "step_download_pdfs": "Downloading PDFs",
    "step_generate_answer": "Generating answer",
    "step_interpret_request": "Interpreting request",
    "step_search_documents": "Searching documents",
    "step_search_passages": "Searching passages",
    "unexpected_error": "An unexpected error occurred. See the log file for details.",
    "unknown_document": "Unknown document",
}

File: frontend/src/locales/en.js
// English UI messages. Keys shared with src/bundesrag/locales/en.py keep the
// same name and wording there so drift between the catalogs is easy to grep for.
export default {
  answer_submit: 'Answer',
  ask_download_count:
    '{num_matched} documents found, {num_existing} of them already downloaded. ' +
    'How many of the remaining {num_to_download} should be downloaded ' +
    '(most recent first, 0 to cancel)?',
  ask_placeholder: 'Which legislative projects exist regarding artificial intelligence?',
  ask_question_label: 'Question about the indexed documents',
  ask_submit: 'Ask question',
  ask_title: 'Ask',
  cancel_requested: 'Cancellation requested …',
  cancel_submit: 'Cancel',
  clear_description:
    'Deletes all downloaded PDFs, resets the vector store, and empties the list of pending ' +
    'documents. This step cannot be undone.',
  clear_submit: 'Delete everything',
  clear_title: 'Clear',
  confirm_delete_all: 'Really delete all downloaded documents and reset the vector store?',
  confirm_delete_file:
    'Really delete file “{file}” (including its entries in the vector store)?',
  confirm_use_query: 'Use this query?',
  count_submit: 'Confirm',
  delete_done: 'Done: {num_files} files deleted and vector store reset.',
  delete_file_label: 'Delete file',
  download_done: 'Done: {num_documents} documents downloaded.',
  download_partial_failure:
    'Warning: {num_failed} document(s) could not be downloaded and were skipped.',
  download_prompt_label: 'Description of the documents to fetch',
  download_prompt_placeholder: 'Plenary protocols of the 21st electoral term.',
  download_running: 'Processing …',
  download_skipped_existing:
    'Note: {num_skipped} document(s) were already downloaded and were skipped.',
  download_submit: 'Build query',
  download_title: 'Download',
  error_prefix: 'Error: {error}',
  index_counts: '{num_to_index} document(s) to index, {num_indexed} already indexed.',
  index_description:
    'Splits all downloaded, not yet indexed documents into chunks and stores them in the ' +
    'vector store. Depending on the volume, this can take a while.',
  index_done: 'Done: {num_documents} documents, {num_chunks} chunks stored.',
  index_running: 'Indexing …',
  index_submit: 'Start indexing',
  index_title: 'Index',
  kind_drucksache: 'Drucksache',
  kind_plenarprotokoll: 'Plenary protocol',
  nav_ask: 'Ask',
  nav_clear: 'Clear',
  nav_download: 'Download',
  nav_index: 'Index',
  nav_status: 'Status',
  no: 'No',
  operation_cancelled: 'Operation cancelled.',
  progress_count: '{current} of {total}',
  source_show_text: 'Show retrieved text',
  sources_header: 'Sources:',
  status_file_indexed: 'indexed',
  status_file_not_indexed: 'not indexed',
  status_chunk_mismatch:
    'Warning: the vector store contains {num_chunks} chunks, but the document ' +
    'metadata accounts for {num_expected} — e.g. due to an aborted indexing run ' +
    'or data modified outside the tool.',
  status_loading: 'Loading status …',
  status_no_documents: 'No documents downloaded.',
  status_num_chunks: 'Chunks in the vector store: {count}',
  status_num_downloaded: 'Downloaded: {count}',
  status_num_indexed: 'Indexed: {count}',
  status_pdf_size: 'Disk usage of PDFs: {size}',
  status_source_link: 'PDF',
  status_th_actions: 'Actions',
  status_th_chunks: 'Chunks',
  status_th_datum: 'Date',
  status_th_doc_id: 'DIP ID',
  status_th_dokumentnummer: 'Document number',
  status_th_file: 'File',
  status_th_kind: 'Type',
  status_th_pages: 'Pages',
  status_th_source: 'Source',
  status_th_status: 'Status',
  status_th_title: 'Title',
  status_title: 'Status',
  status_vectorstore_size: 'Disk usage of the vector store: {size}',
  yes: 'Yes',
}
